3581/3/app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewController < ApplicationController
+  before '/reviews/' do
+    redirect '/' unless Place.exists?(params[:id])
```

Comment:
better
```ruby
@place = Place.find(params[:id])
```
and remove the same line in the controller
it is the extra query
it raises error which would be handled